.zuul.yaml

```diff
@@ -0,0 +1,9 @@
+---
+- project:
+    name: openstack-k8s-operators/watcher-operator
+    default-branch: main
+    github-check:
+      jobs:
+        - openstack-meta-content-provider:
+            vars:
+              cifmw_operator_build_meta_build: false
```

Comment:
@marios @SeanMooney Thank you for the reviews and comment! you both are correct, `cifmw_operator_build_meta_build` is skipping meta operator build.

Reply:
> only concern/question is do we want to start this yet? 
For right now, we want to go ahead like this. It will validate the building process of watcher operator.

> I mean lets add some jobs that need the content provider ? (or do we already have something lined up right after this merges) Like this it will just build images every time and nothing will use them.

Once some CRDS and Kuttl tests will be added to the operator, then I think we can proceed with adding Kuttl/EDPM job to consume the operator image from content provider job.